src/rebar_utils.erl

```diff
@@ -179,19 +179,6 @@ expand_code_path() ->
                            end, [], code:get_path()),
     code:set_path(lists:reverse(CodePath)).
 
-vcs_vsn(Config, Vsn, Dir) ->
-    Key = {Vsn, Dir},
-    Cache = rebar_state:get(Config, vsn_cache, dict:new()),
```

Comment:
Did you just drop the cache for any specific reason there, or it's just dropping cruft or something?